### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by cala**
Mana regeneration from spirit points
Unstealth when looting
Glancing blow fix

### CODE REVIEW

**Comment by Salja** [comment]
Jepp the mana fix is rigth i have sice 1 year https://github.com/Salja/MaNGOSZero/commit/b8da98d481623cfc01e23ba86de5d8564a1a4143

and the other all ok

File: src/game/Player.cpp [hunk 1]
```diff
@@ -6937,6 +6937,9 @@ void Player::SendLoot(ObjectGuid guid, LootType loot_type)
     PermissionTypes permission = ALL_PERMISSION;
 
     DEBUG_LOG("Player::SendLoot");
+	
+    RemoveSpellsCausingAura(SPELL_AURA_MOD_STEALTH);
```
Comment (Zakamurite) [inline src/game/Player.cpp]: Wrong way. Need use interrupt flag

**Comment by cala** [comment]
@Salja : my apologies, I didn't know who was the original author of this update I fetched from Picard64 repository.

File: src/game/Player.cpp (hunk 1 above)
Comment (cala) [inline src/game/Player.cpp]: Won't using interrupt flag also interrupt the player from looting corpse?
If not I will look how flag system work and try to do it.

**Comment by Salja** [comment]
@cala it does not matter=) its fine wehn ppls post fixes for zero =) look in my repo i think you found some more good patches

**Comment by cala** [comment]
@Salja ok, thank you. I'll look to your repo. =) I submitted those three ones because I was sure they were blizz-like.

File: src/game/Player.cpp (hunk 1 above)
Comment (cala) [inline src/game/Player.cpp]: Okay, I looked at the 25 available INTERRUPT_FLAG and none of them seem to be for loot. I understand what you suggest but everywhere else, stealth is removed this way: RemoveSpellsCausingAura(SPELL_AURA_MOD_STEALTH);
If you know which INTERRUPT_FLAG to use, I'll try it.

**Comment by KillaVulkan** [comment]
Yes, Salja repo is best!!!!!!
I used for my server :P

**Pull request merged**